pkg/client/client.go

```diff
@@ -52,7 +53,7 @@ type OptimizelyClient struct {
 // CreateUserContext creates a context of the user for which decision APIs will be called.
 // A user context will be created successfully even when the SDK is not fully configured yet.
 func (o *OptimizelyClient) CreateUserContext(userID string, attributes map[string]interface{}) OptimizelyUserContext {
-	return newOptimizelyUserContext(o, userID, attributes)
+	return newOptimizelyUserContext(o, userID, attributes, nil)
```

Comment:
Since we don't have any instance of forcedDecisionService to pass since a new `userContext` is being created. In this case, `newOptimizelyUserContext` will create a `forcedDecisionService` internally by itself .